liquibase/src/test/java/org/openmrs/liquibase/AbstractSnapshotTunerTest.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.liquibase;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalToCompressingWhiteSpace;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class AbstractSnapshotTunerTest {
+	
+	private static final String FILE_WITH_LICENSE_HEADER_MD = "file-with-license-header.md";
+	
+	private static final String FILE_WITHOUT_LICENSE_HEADER_MD = "file-without-license-header.md";
+	
+	private static final String HTTP_OPENMRS_ORG_LICENSE = "http://openmrs.org/license";
+	
+	private static String PATH_TO_TEST_RESOURCES = "src" + File.separator + "test" + File.separator + "resources"
+	        + File.separator;
+	
+	/*
+	 * An instance of org.openmrs.liquibase.SchemaOnlyTuner is used to test behaviour implemented in the 
+	 * org.openmrs.liquibase.AbstractSnapshotTuner class.
+	 */
+	private SchemaOnlyTuner schemaOnlyTuner;
+	
+	@Rule
+	public TemporaryFolder tempFolder = new TemporaryFolder();
```

Comment:
this will need to be replaced with JUnit 5 https://junit.org/junit5/docs/5.4.0/api/org/junit/jupiter/api/io/TempDir.html

Reply:
Done.